### PR TITLE
Handling RTL

### LINKED ISSUE #26: Handle RTL when switching to Farsi

### PR DESCRIPTION
**Pull request by AmirSavand**
Fixes #26

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -2,10 +2,10 @@
   "DefaultContentLanguage": "en",
   "languages": {
     "en": {
-      "baseURL": "https://savandbros.com/en",
+      "baseURL": "https://savandbros.com",
       "languageName": "English",
       "title": "English",
-      "weight": 0
+      "weight": 2
```
Comment (AmirSavand) [inline config.json]: English was 0 in the docs of hugo.

File: layouts/index.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" dir="{{ if eq .Language.Lang "fa" }}rtl{{ else }}ltr{{ end }}">
```
Comment (AmirSavand) [inline layouts/index.html]: I found an unanswered question about RTL and I got here.

File: static/assets/app.css [hunk 3]
```diff
@@ -13,6 +13,19 @@ a .border:hover {
 }
 
 
+/* Dir Style */
```
Comment (AmirSavand) [inline static/assets/app.css]: Some sections didn't work with RTL

File: static/assets/app.css [hunk 4]
```diff
@@ -36,6 +49,29 @@ a .border:hover {
 }
 
 
+/* Footer */
```
Comment (AmirSavand) [inline static/assets/app.css]: All the default styles are now here instead of classes.

File: layouts/index.html (hunk 2 above)
Comment (Alir3z4) [reply layouts/index.html]: Nice.

File: static/assets/app.css (hunk 3 above)
Comment (Alir3z4) [reply static/assets/app.css]: Yeah, RTL can be tricky in some cases

File: static/assets/app.css (hunk 4 above)
Comment (Alir3z4) [reply static/assets/app.css]: Good job on this one.

**Comment by Alir3z4** [comment]
We should definitely take the lesson learned from here to GoNevis i18n later as well.
Although our tools in GoNevis are much more flexible and mature to work with.

**Comment by AmirSavand** [comment]
Yep

**Pull request merged**